<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gpbBusca">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>168</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnBuscar">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLimpar">
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>41</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblNome">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNome"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>41</y>
       <width>177</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblDepartamento">
        <property name="text">
         <string>Departamento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbDepartamento">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableView" name="tblTabela">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>641</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastros">
    <property name="title">
     <string>Cadastros</string>
    </property>
    <addaction name="actionDepartamentos"/>
    <addaction name="actionPessoasETelefones"/>
    <addaction name="actionFechar"/>
   </widget>
   <widget class="QMenu" name="menuPesquisas">
    <property name="title">
     <string>Pesquisas</string>
    </property>
   </widget>
   <addaction name="menuCadastros"/>
   <addaction name="menuPesquisas"/>
  </widget>
  <widget class="QToolBar" name="tlbToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDepartamentos">
   <property name="icon">
    <iconset resource="recursos_imagens.qrc">
     <normaloff>:/Imagens/Icones/my-profile.png</normaloff>:/Imagens/Icones/my-profile.png</iconset>
   </property>
   <property name="text">
    <string>Departamentos</string>
   </property>
  </action>
  <action name="actionPessoasETelefones">
   <property name="icon">
    <iconset resource="recursos_imagens.qrc">
     <normaloff>:/Imagens/Icones/receptionist.png</normaloff>:/Imagens/Icones/receptionist.png</iconset>
   </property>
   <property name="text">
    <string>Pessoas e Telefones</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="icon">
    <iconset resource="recursos_imagens.qrc">
     <normaloff>:/Imagens/Icones/click-to-call.png</normaloff>:/Imagens/Icones/click-to-call.png</iconset>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos_imagens.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>txtNome</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
